packages/manager/modules/nutanix/src/dashboard/general-info/routing.js

```diff
@@ -3,8 +3,6 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '',
     component: 'nutanixGeneralInfo',
```

Comment:
related to this:
https://github.com/ovh/manager/pull/5830/files#r742619275

Reply:
Change not related to this PR. Need to do a refactoring to make this change. It need to be done as part of a new PR.